src/helpers/a11yAPI.js

```diff
@@ -58,6 +69,17 @@ export function matchArrayValue(
   return !matcher.some(e => !prop.includes(e));
 }
 
+export function matchObject<T: {}>(prop?: T, matcher: T): boolean {
+  return (
+    Boolean(prop) &&
+    Object.keys(matcher).length !== 0 &&
+    // $FlowFixMe Already checked
+    Object.keys(prop).length !== 0 &&
+    // $FlowFixMe Already checked
+    !Object.keys(matcher).some(key => prop[key] !== matcher[key])
+  );
```

Comment:
This would get rid of fixmes
```suggestion
  return prop
    ? Object.keys(matcher).length !== 0 &&
        Object.keys(prop).length !== 0 &&
        !Object.keys(matcher).some(key => prop[key] !== matcher[key])
    : false;
```